<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="10" column="1">
    <widget class="QListWidget" name="listwidget_other">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QListWidget" name="listwidget_spx">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="checkbox_vtk">
     <property name="text">
      <string>VTK files (*.vtp, *.vtu, *.pvd, *.pvtp, *.pvtu)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="checkbox_spx">
     <property name="text">
      <string>SPx files (*.SP?)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="checkbox_res">
     <property name="text">
      <string>Restart/mesh files (*.RES, BACKUP_RES/*.RES*, *.msh)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QListWidget" name="listwidget_restart">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_heading">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delete the following files?</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QCheckBox" name="checkbox_other">
     <property name="text">
      <string>Other (*.OUT, *.pid, *.error, *.e[0-9]*, *.0[0-9]*, *.pe[0-9]*, *.po[0-9]*)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QListWidget" name="listwidget_vtk">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="checkbox_monitor">
     <property name="text">
      <string>Monitor and log files (*.csv)</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QListWidget" name="listwidget_monitor"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
